### PR TITLE
=doc #16672 explain relationship between AS and RS

### LINKED ISSUE #16672: doc: make it more prominent that Reactive Streams has nothing to do with Akka Streams API

### PR DESCRIPTION
**Pull request by rkuhn**
closes #16672

### CODE REVIEW

File: akka-docs-dev/rst/java/stream-introduction.rst [hunk 1]
```diff
@@ -45,6 +45,21 @@ implementations like Akka Streams offer a nice user API).
...
+----------------------------------
+
+The Akka Streams API is completely decoupled from the Reactive Streams
+interfaces. While Akka Streams focus on the formulation of transformations on
+data streams the scope of Reactive Streams is just to define how to move data
+across an asynchronous boundary without losses, buffering or resource
+exhaustion.
+
+The relationship between these two is that the Akka Streams API is geared
+towards end-users while the Akka Streams implementation uses the Reactive
```
Comment (ktoso) [inline akka-docs-dev/rst/java/stream-introduction.rst]: the `while` here reads a bit disconnected. "and while" perhaps?

File: akka-docs-dev/rst/java/stream-introduction.rst [hunk 2]
```diff
@@ -45,6 +45,21 @@ implementations like Akka Streams offer a nice user API).
...
+Streams interfaces internally to pass data between the different processing
+stages. For this reason you will not find any resemblance between the Reactive
+Streams interfaces and the Akka Streams API.
```
Comment (patriknw) [inline akka-docs-dev/rst/java/stream-introduction.rst]: shouldn't there be something about that RS is about integration between different RS implementations, where Akka Streams provides one such implementation

File: akka-docs-dev/rst/java/stream-introduction.rst [hunk 3]
```diff
@@ -45,6 +45,21 @@ implementations like Akka Streams offer a nice user API).
 
 .. _Reactive Streams: http://reactive-streams.org/
 
+Relationship with Reactive Streams
+----------------------------------
+
+The Akka Streams API is completely decoupled from the Reactive Streams
+interfaces. While Akka Streams focus on the formulation of transformations on
+data streams the scope of Reactive Streams is just to define how to move data
+across an asynchronous boundary without losses, buffering or resource
```
Comment (drewhk) [inline akka-docs-dev/rst/java/stream-introduction.rst]: while that is true, the point is interoperability, to have a standard interface for such boundaries.

**Comment by drewhk** [comment]
LGTM

File: akka-docs-dev/rst/java/stream-introduction.rst (hunk 1 above)
Comment (rkuhn) [inline akka-docs-dev/rst/java/stream-introduction.rst]: no, “and while” would not be correct grammar: this is a case switch

**Pull request merged**